domain-ee/ee-ep-merge-app/src/python_src/service/ep_merge_machine.py

```diff
@@ -40,6 +40,9 @@
 JOB_SKIPPED_MERGE_METRIC = 'job.skipped_merge'
 JOB_NEW_CONTENTIONS_METRIC = 'job.new_contentions'
 
+EP400_PRODUCT_CODES = frozenset([str(i) for i in range(400, 410)])
+EP400_BENEFIT_CLAIM_TYPE_CODES = frozenset(['400SUPP'])
```

Comment:
I like this. If you want to be even more clever, you can just do `EP400_PRODUCT_CODES = range(400, 410)` -- range objects are magical lightweight stand-ins for their more explicit counterparts.

Reply:
Ah nice, but need it to be strings!

Reply:
Ahaha, of course. 👍